utils/format_response.py

```diff
@@ -1,9 +1,16 @@
 import json
+from http import HTTPStatus
 
 
 def format_response(statusCode, message, data=None):
-
-    body = {"statusCode": statusCode, "data": data, "message": message}
+    if HTTPStatus.OK.value >= statusCode and statusCode <= HTTPStatus.ALREADY_REPORTED.value:
```

Comment:
Lets have consistent message formats. For success we are using the key "statusCode" but for errors its "status"

Reply:
Changed the statusCode to status in success in the latest commit